codeexamples/SYCLUsingCUDAStaticLibraryOfKernels/StaticLibraryBuildCUDAKernels/.vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,18 @@
+{
+    "configurations": [
+        {
+            "name": "Linux",
+            "includePath": [
+                "${workspaceFolder}/**",
+                "${workspaceFolder}/../../cuda-samples-nvidia/Common",
+                "/usr/local/cuda-12.0/targets/x86_64-linux/include"
```

Comment:
I don't think we should be hardcoding this path (for example, CUDA is definitely not installed there on my development machine).

Reply:
As I said to Rafal, these configuration files are required for the project to work. Even if incorrect for a different user they can inform the user on how to edit the config to best for their environment.   Hard paths are never intentional, but there isn't an alternative I know to do this.

Reply:
Yes required for Intellisense to operate correctly for the project. If Intellisense if not configured for a project, the VSCode IDE will display underscore squiggles through out the code to show it does not understand. This file is required.